coverage/clover.xml

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
shouldn't coverage reports be in .gitignore?

Reply:
added